package-lock.json

```diff
@@ -1955,7 +1955,7 @@
     },
     "create-hash": {
       "version": "1.2.0",
-      "resolved": "http://registry.npmjs.org/create-hash/-/create-hash-1.2.0.tgz",
+      "resolved": "https://registry.npmjs.org/create-hash/-/create-hash-1.2.0.tgz",
```

Comment:
Why are there changes to the lock but not the `package` file?

Reply:
I did a `npm install` after pulling from master for the `wildcard` babel plugin. Should I delete my `package-lock.json` and then reinstall?